practice_flow_control_and_methods/last_letter_matcher.rb

```diff
@@ -1 +1,32 @@
-# Ask for two words. Say whether or not they end in the same letter.
\ No newline at end of file
+# Ask for two words. Say whether or not they end in the same letter.
+
+def get_words
+  puts 'word one:'
+  word_one = gets.chomp.downcase
+  puts 'word two:'
+  word_two = gets.chomp.downcase
+  return word_one, word_two
```

Comment:
`A Ruby method technically returns only one object.`

but here, youre returning two. if that's true then how is this working? why didnt you get an error?